src/index.html

```diff
@@ -22,6 +22,468 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="smartphones"
```

Comment:
The `href` attribute for 'Smartphones' seems to be missing the leading slash, which might cause the link to be relative to the current page path. Ensure it's an absolute path if that's the intention.